docs/content/preview/yugabyte-platform/back-up-restore-universes/restore-universe-data/ycql.md

```diff
@@ -30,29 +30,68 @@ showAsideToc: true
 
 </ul>
 
+## Basic restore procedure
+
 You can restore YugabyteDB universe YCQL data from a backup as follows:
 
-1. Open your universe and then select **Backups**.
+1. Open your universe and select **Backups**.
+
+2. In the **Backups** page, select the backup you want to restore and click its **... > Restore Entire Backup**, as per the following illustration:<br><br>
```

Comment:
When we click on the backup, we list down all the keyspaces that were backed up. We can also select a keyspace from this list for restoring.

Reply:
All the steps mentioned for the entire backup restore are applicable here too.

Reply:
Added a step for a keyspace restore